### ISSUE TITLE
About the small text above the other of some novels

### ISSUE THREAD

**Issue by LeviTheli** [dteviot/WebToEpub#113]
Hello, devs
I want to seperate the small text with brackets.
I'm not sure if this have been solved here or not but I'm looking for a way to solve this but haven't found any.
e.g. like this series
https://www.baka-tsuki.org/project/index.php?title=Magika_No_Kenshi_To_Shoukan_Maou
![ori](https://cloud.githubusercontent.com/assets/27391093/24986771/625e4afc-2026-11e7-95b3-2403456fa5b4.jpg)

**Here is the epub I downloaded from Baka-tsuki ePUB Generator (Volume 11)**
![vol11](https://cloud.githubusercontent.com/assets/27391093/24986685/eec175f6-2025-11e7-953c-fcd18d5e7e70.jpg)

**And here is the epub I made with WebToEpub**
![vol12](https://cloud.githubusercontent.com/assets/27391093/24986811/9d4f5bce-2026-11e7-922d-87d3b4fca436.jpg)

I think it's a stylesheet problem but I'm ignorance with it so I'm here asking for help.
Thank you for reading this.

**Comment by dteviot** [comment]
@LeviTheli 

It's not a stylesheet problem.
If you look at the chapter heading for book 12, chapter 5, the HTML in the document is:
```html
    <span class="mw-headline" id="Chapter_5_.E2.80.93_ZeusLightning_Thunder_and_OdinSeverance_Spear.2C_Twisted_Dragon_of_Destiny">Chapter 5 – 
        <span style="white-space: nowrap; position: relative;">
            <span style="position: absolute; font-size: .8em; top: -11px; left: 50%; white-space: nowrap; letter-spacing: normal; color: inherit; font-weight: inherit; font-style: inherit;">
                <span style="position: relative; left: -50%;">Zeus</span>
            </span>
            <span style="display: inline-block; color: inherit; letter-spacing: normal; font-size: 1.0em; font-weight: inherit;">Lightning Thunder</span>
        </span> and 
        <span style="white-space: nowrap; position: relative;">
            <span style="position: absolute; font-size: .8em; top: -11px; left: 50%; white-space: nowrap; letter-spacing: normal; color: inherit; font-weight: inherit; font-style: inherit;">
                <span style="position: relative; left: -50%;">Odin</span>
            </span>
            <span style="display: inline-block; color: inherit; letter-spacing: normal; font-size: 1.0em; font-weight: inherit;">Severance Spear</span>
        </span>, Twisted Dragon of Destiny
    </span>
```
 As you may be able to see, &lt;span&gt; elements have been used to move "Zeus" and "Odin" above the "Lightning Thunder" and "Severance Spear" text elements respectively.   However, when the plug-in collects the text for the chapter title, it just reads the text in order and does not allow for the &lt;span&gt; elements.  

So, the parser needs to be made smarter when collecting chapter titles for the epub's table of contents.  It must
1. Look for these sorts of &lt;span&gt; elements  (Might not be difficult, probably just needs to look a "top" attribute with a negative value.)
2. When found, enclose the text in brackets, and
3. Move the text to after the following &lt;span&gt; element.

**Comment by LeviTheli** [comment]
Thank you for replying
Well, It's not only the chapter titles but also in the chapter too.
So I guess I would wait until someone could include this function into the WebToEpub since I know nothing about programming.

**Comment by dteviot** [comment]
@LeviTheli 
I've added a new "Remove superscript from alternate translations" option to "Advanced Settings".
If checked, it will move all "superscripted" text (not just in chapter titles) as described above.
This is currently in the Experimental branch, so you'll need to go to https://github.com/dteviot/WebToEpub/tree/ExperimentalTabMode
and follow the instructions in the "How to install from Source" section.  **Note, you'll need to download the code from https://github.com/dteviot/WebToEpub/tree/ExperimentalTabMode, not https://github.com/dteviot/WebToEpub**
Please let me know how it works.

**Comment by toshiya44** [comment]
It works.
![image](https://cloud.githubusercontent.com/assets/20046789/25214282/63b6cf76-2564-11e7-95f2-836f18041553.png)

**Comment by LeviTheli** [comment]
@dteviot 
Thank you very much, sir
So far so good, I'll test it further and let's u know if something happens.

**Comment by dteviot** [comment]
@toshiya44, @LeviTheli 
Thank you both for the update.  As you may have noticed, this code has been included in the latest update (0.0.0.34) to the Chrome store.
